mlflow/entities/model_registry/registered_model.py

```diff
@@ -48,6 +50,12 @@ def latest_versions(self):
         for each stage"""
         return self._latest_version
 
+    @property
+    def tags(self):
+        """List of :py:class:`mlflow.entities.model_registry.RegisteredModelTag` instances
```

Comment:
@zhidongqu-db For consistency with Run Tags and Experiment Tags, this should be a dictionary of key:value pairs, rather than a list. See https://github.com/mlflow/mlflow/blob/master/mlflow/entities/run_data.py#L42